forms/pdf_form_fields_rotations.go

```diff
@@ -0,0 +1,139 @@
+/*
+ * Get form fields appearance rotation value.
```

Comment:
hm, what is the relation to images then?

Reply:
Updated at e3c009fe45cbb4df9bd4d909958454d04997382d